dask/core.py

```diff
@@ -289,7 +289,7 @@ def subs(task, key, val):
         try:
             if task == key:
                 return val
-        except ValueError:
+        except:
```

Comment:
This will also catch `SystemExit` and `KeyboardInterrupt`.  I would recommend `except Exception:` instead.  Seems like a reasonable change to do, since we don't know what values the user will use, so we don't know what exceptions may be raised.